src/main/java/org/refactoringminer/api/RefactoringType.java

```diff
@@ -229,10 +229,10 @@ public enum RefactoringType {
 		TRY_WITH_RESOURCES,
 		REPLACE_GENERIC_WITH_DIAMOND,
 		REPLACE_CONDITIONAL_WITH_TERNARY,
-		REPLACE_CONDITIONAL_WITH_TERNARY,
 		REPLACE_CONDITIONAL_WITH_ASSUMPTION,
 		REPLACE_ASSERTION_WITH_ASSUMPTION,
-		REPLACE_IGNORE_WITH_ASSUMPTION
+		REPLACE_IGNORE_WITH_ASSUMPTION,
+        EXTRACT_FIXTURE
```

Comment:
The Extract Fixture description should include the target fixture
`EXTRACT_FIXTURE("Extract Test Fixture", "Extract Test Fixture (.+) extracted from (.+) in class (.+)");`
similar to `EXTRACT_OPERATION`

`EXTRACT_OPERATION("Extract Method", "Extract Method (.+) extracted from (.+) in class (.+)", 2)`

The new refactoring description format most likely makes some tests depending on refactoring descriptions (`TestAllRefactoringsByCommit`, `TestNewDatasetRefactorings`) to fail.

The change is not so innocent as it looks. I will need to update the refactoring descriptions in the benchmarks for the tests to pass.